### PR TITLE
Update Base_German.lang

### PR DESCRIPTION
**Pull request by daspositioned**

### CODE REVIEW

File: Translations/Base_German.lang [hunk 1]
```diff
@@ -23,10 +23,10 @@ GUI_SETTINGS_MYSQL_RESTARTREQUIRED: Ein Restart wird nach der Einrichtung benöt
 GUI_SETTINGS_MYSQL_CANTCONNECT: Konnte nicht mit MySQL verbinden
 GUI_SETTINGS_MYSQL_SAVE_TITLE: Überprüfen & Speichern
 GUI_SETTINGS_MYSQL_SAVE_ACTION: Klicke um zu **überprüfen** und zu **speichern**
-#GUI_SETTINGS_LANG_ACTION: Click to **select **this language
-#GUI_SETTINGS_LANG_SELECTED: This language is **currently** selected
-#GUI_SETTINGS_LANG_PHRASES: Phrases
-#GUI_SETTINGS_LANG_COVERAGE: Coverage
+#GUI_SETTINGS_LANG_ACTION: Klicke um diese Sprache **auszuwählen**
+#GUI_SETTINGS_LANG_SELECTED: Diese Sprache wurde **aktuell** ausgewählt
```
Comment (rlnt) [inline Translations/Base_German.lang]: Please change to "Diese Sprache ist **derzeit** ausgewählt". That is fitting more to the style of the rest.

**Review by rlnt — CHANGES_REQUESTED** [review]

**Comment by Seliba** [comment]
Updated translations

**Comment by daspositioned** [comment]
Updated translations

**Comment by DAmNRelentless** [comment]
The entry is doubled now. Go to the patch-2 branch at your forked repository and delete the old line.

**Comment by daspositioned** [comment]
Fixed, sorry for that

**Comment by rlnt** [comment]
Thanks for your submission!

**Pull request merged**